projects/developer/src/app/common/components/recipe-graph/component.ts

```diff
@@ -1180,4 +1180,17 @@ export class RecipeGraphComponent implements OnInit, OnChanges, AfterViewInit {
             this.update.next(true);
         }, 0);
     }
+
+    public hasConnection(filename, connections) {
+        const file_name = filename;

```

Comment:
No need to reassign the function params as new variables.

Reply:
OK.